LeanKit-Labs/tag-release
# Added --pr feature to update upstream repo with package.json changes and create a PR to develop for change

### Short description of the work completed

> Added `--pr` feature to promote a `--qa` create branch that includes changes to the `package.json` into release ready mergeable PR against develop for `web-lightning-ui` and `core-leankit-api`.

### Steps to test (if not obvious)

1. Checkout `develop`
2. run `tag-release --qa` and follow flow to update some package versions.
3. run `tag-release --pr` from branch created from `--qa` flow and follow steps
4. should now have a mergeable PR against develop with the real versions you wish to change these packages to.

NOTE: You can also provide your own scope to these flags.

### For Reviewer Use Only

  patch)
